comp/core/autodiscovery/autodiscoveryimpl/autoconfig.go

```diff
@@ -94,6 +96,15 @@ type AutoConfig struct {
 	ranOnce *atomic.Bool
 }
 
+const (
+	// MaxWmetaWaitTime is the maximum time to wait for wmeta being ready
+	MaxWmetaWaitTime = 10 * time.Second
+	// WmetaCheckMinInterval is the initial interval to check if wmeta is ready
+	WmetaCheckMinInterval = 200 * time.Microsecond
```

Comment:
What do you think about having the initial internal value being a bit higher? Maybe more like `20 * time.Millisecond`?

Reply:
```suggestion
	WmetaCheckMinInterval = 20 * time.Millisecond
```